### PR TITLE
Started writing some basic tests

### PR DESCRIPTION
**Pull request by ranajaydas**
The test are using pytest and have a detailed coverage report to ensure **everything** is tested :3.

I mean, if we're doing this, might as well do it right 😎.

Btw, the mixer package is amazing for quickly creating database objects. It even does edge case testing because of the randomized inputs.

### CODE REVIEW

File: otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def test_function_extracts_correct_data_from_page_with_hidden_redirect(self):
 
 class TestParseSubjectList(TestCase):
```
Comment (metakermit) [inline otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py]: This is good. I suggest you take a look how much easier writing tests as functions with pytest can be now that you've done it "the classical way" https://docs.pytest.org/en/latest/getting-started.html#getstarted

File: otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py [hunk 2]
```diff
@@ -112,5 +112,6 @@ def test_function_saves_all_data(self):
 
         self.assertEqual(Period.objects.count(), 1)
         self.assertEqual(Item.objects.count(), 17)
-        self.assertEqual(Subject.objects.count(), 19)
-        self.assertEqual(Act.objects.count(), 105)
+        self.assertEqual(Subject.objects.count(), 25)
```
Comment (metakermit) [inline otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py]: In pytest you could use Python's built in `assert` keyword:

```
assert Art.objects.count() == 105
```

File: otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py (hunk 1 above)
Comment (ranajaydas) [reply otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py]: I like using the Django TestCase. I can try direct asserts too but without pytest they aren't that informative so I avoided using them.

Comment (metakermit) [reply otvoreni_akti/apps/scraper/tests/test_scrape_utils_html.py]: OK, that's perfectly fine of course. Just thought I'd share it with you 🙂

**Comment by metakermit** [comment]
Wow, awesome! 💪 

I love it how in the end I'm actually learning about new packages from you! 🙂

**Comment by ranajaydas** [comment]
:D ❤️
